src/AutoMapper/PropertyMap.cs

```diff
@@ -41,7 +41,7 @@ public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
         public LambdaExpression CustomExpression { get; private set; }
         public MemberInfo CustomSourceMember { get; set; }
         public bool UseDestinationValue { get; set; }
-        public bool ExplicitExpansion { get; set; }
+        public bool ExplicitExpansion { get { return _explicitExpansion.GetValueOrDefault(); } set { _explicitExpansion = value; } } private bool? _explicitExpansion; 
```

Comment:
@jbogard, I think that ExplicitExpansion and other bool properties listed in the link below need to be non-nullable booeans, but backing fields have to be nullable for including base configuration sake.

The list of bool mapping config properties is @ SO answer by Ivan Stoev: http://stackoverflow.com/questions/41945809/automapper-5-2-ignores-explicitexpansion-if-it-is-configured-in-base-dto-mapping/42012290#42012290

The coding offer to introduce nullable backing field in below in the Ivan Stoev's comment to this answer: http://stackoverflow.com/questions/41945809/automapper-5-2-ignores-explicitexpansion-if-it-is-configured-in-base-dto-mapping/42012290#comment71390369_42012290